site/src/components/Playground.js

```diff
@@ -137,14 +140,17 @@ const Editor = ({ onChange, initialCode, code, language }) => {
       const textArea = getTextArea(viewPortRef.current);
       textArea.focus();
       textArea.select();
+      setCopyState('');
 
       try {
         document.execCommand('copy');
```

Comment:
Off-topic from the PR, Intellij IDEA warns about document.execCommandy being deprecated:

<img width="725" alt="image" src="https://user-images.githubusercontent.com/2777633/196680465-20f5dc03-e50f-4a67-b1c9-30e5a98c32b4.png">

Reply:
I'll check if it would be an easy to change 👍